extensions/amp-story/amp-story-page-attachment.md

```diff
@@ -59,32 +59,109 @@ A UI prompt to open the attachment will automatically be added at the bottom of
 </amp-story-page>
 ```
 
-### Placement
+## Placement
 
 The `<amp-story-page-attachment>` element must be the last child of `<amp-story-page>`, and must have the `layout="nodisplay"` attribute. The attachment AMPHTML content is expected to be provided inline in your AMP Story, within this `<amp-story-page-attachment>` tag.
 
 [tip type="important"]
-Both [`amp-story-cta-layer`](amp-story-cta-layer.md) and `amp-story-page-attachment` must be the last child tag of an [`amp-story-page`](amp-story-page.md). Because of this, you may include neither or one, but not both.
+Both [`amp-story-page-attachment`](amp-story-page-attachment.md) and [`amp-story-page-outlink`](amp-story-page-outlink.md) must be the last child tag of an [`amp-story-page`](amp-story-page.md). Because of this, you may include neither or one, but not both.
 [/tip]
 
-### Linked content
+## Attributes
+
+When no attributes are set, the default UI will render:
+
+![amp-story-page-attachment-default](img/amp-story-page-attachment-default.jpg)
+
+```html
+<amp-story-page-attachment
+  layout="nodisplay">
+   ...
+</amp-story-page-attachment>
+```
+
+### `theme` (optional)
+
+String representing the color theme, default is `light`. Accepts values `light` & `dark`.
+
+![amp-story-page-attachment-dark-theme](img/amp-story-page-attachment-dark-theme.jpg)
+
+```html
+<amp-story-page-attachment
+  layout="nodisplay"
+  theme="dark">
+  ...
+</amp-story-page-attachment>
+```
+
+### `cta-text` (optional)
+
+String that customizes the button text. The default is "Swipe up".
+
+![amp-story-page-attachment-cta-text](img/amp-story-page-attachment-cta-text.jpg)
+
+```html
+<amp-story-page-attachment
+  layout="nodisplay"
+  cta-text="Call To Action">
+   ...
+</amp-story-page-attachment>
+```
+
+### `cta-image` (optional)
+
+String representing a URL pointing to an image. 48x48px is suggested.
```

Comment:
I'm a little stuck on the word "suggested" - will other sizes work? I would update the language to something like "Optimize performance and avoid distortion by using a 48x48px image"